### PR TITLE
fix(python): template cleanup and playgrounds

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1667

### Changes included:

follow up of https://github.com/algolia/api-clients-automation/pull/2283, initially done in https://github.com/algolia/api-clients-automation/pull/2285

This PR removes some of the unused stuff of the templates (librairies, imports, some debugging methods etc.)

Also adds the playground, which allowed me to properly setup the imports.

The next PR will focus on the requester using aiohttp for asynchronous calls.

### CODE REVIEW

File: clients/algoliasearch-client-python/setup.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import sys
+
+if sys.version_info < (3, 8):
```
Comment (millotp) [inline clients/algoliasearch-client-python/setup.py:3]: this comparaison works ? python is too powerful

File: scripts/playground.ts [hunk 2]
```diff
@@ -41,6 +41,9 @@ export async function playground({
       // eslint-disable-next-line no-console
       console.log(await run(`cd playground/go && /usr/local/go/bin/go run . --client ${client}`));
       break;
+    case 'python':
+      await run(`cd playground/python && poetry install && poetry run ${client}`);
```
Comment (millotp) [inline scripts/playground.ts:45]: maybe we should use `cwd` instead of `cd`, for the others too

File: templates/python/partial_header.mustache [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# coding: utf-8
+
 """
     {{{generationBanner}}}
```
Comment (millotp) [inline templates/python/partial_header.mustache:4]: I think this should not be indented

File: clients/algoliasearch-client-python/pyproject.toml
Comment (millotp) [inline clients/algoliasearch-client-python/pyproject.toml:1]: this file should be deleted since it's generated

**Review by millotp — COMMENTED** [review]
beautiful cleanup ! I'll look at the generated code in the next review

File: clients/algoliasearch-client-python/setup.py (hunk 1 above)
Comment (shortcuts) [reply clients/algoliasearch-client-python/setup.py:3]: Yup that's a cute syntax

File: scripts/playground.ts (hunk 2 above)
Comment (shortcuts) [reply scripts/playground.ts:45]: True!

File: clients/algoliasearch-client-python/pyproject.toml
Comment (shortcuts) [reply clients/algoliasearch-client-python/pyproject.toml:1]: good point

**Review by millotp — APPROVED** [review]
that's very nice

**Pull request merged**